packages/javascript-sdk/src/fr-webauthn/index.ts

```diff
@@ -176,7 +182,11 @@ abstract class FRWebAuthn {
         hiddenCallback.setInputValue(`${WebAuthnOutcome.Error}::${error.name}:${error.message}`);
         throw error;
       }
-      hiddenCallback.setInputValue(deviceName.length > 0 ? `${outcome}::${deviceName}` : outcome);
+      if (outcome !== undefined) {
```

Comment:
again, technically outcome shouldn't be undefined but typescript strictness led down this path

Reply:
Would returning in the conditional block above help?

Reply:
I think i can remove this now